tests.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const { describe } = require("yargs");
+
+describe('pick-up' () => {
+    
+}
```

Comment:
If you could add this to a folder called __tests__ that would be really great for file structure!